src/main/scala/org/apache/spark/deploy/armada/submit/ArmadaClientApplication.scala

```diff
@@ -776,12 +867,15 @@ private[spark] class ArmadaClientApplication extends SparkApplication {
       jobSetId: String,
       executors: Seq[api.submit.JobSubmitRequestItem]
   ): Seq[String] = {
-    val executorsResponse = armadaClient.submitJobs(queue, jobSetId, executors)
-    executorsResponse.jobResponseItems.map { item =>
-      val error = Some(item.error).filter(_.nonEmpty).getOrElse("none")
-      log(s"Submitted executor job with ID: ${item.jobId}, Error: $error")
```

Comment:
don't we still want to report errors here?

Reply:
accident